charts/rasa-x/templates/_rasax.nginx.conf.tpl

```diff
@@ -9,40 +10,43 @@ server {
   keepalive_timeout   30;
   client_max_body_size 800M;
 
-  location /robots.txt {
+  location {{ trimSuffix "/" .Values.nginx.subPath }}/robots.txt {
     return 200 "User-agent: *\nDisallow: /\n";
   }
 
-  location /core/ {
+{{- if or .Values.rasa.versions.rasaProduction.enabled .Values.rasa.versions.rasaProduction.external.enabled }}
+  location {{ trimSuffix "/" .Values.nginx.subPath }}/core/ {
     proxy_set_header X-Real-IP $remote_addr;
     proxy_set_header X-Forwarded-For $remote_addr;
     proxy_set_header X-Forwarded-Proto $scheme;
     proxy_set_header Host $host;
-    proxy_pass {{ include "rasa-bot.production.url" .}};
+    proxy_pass {{ include "rasa.production.url" .}};
```

Comment:
This solves adding a subpath if that's the only change made, but what if the user wants to just mount a custom SSL conf, or make another change to the Nginx config? 
Can we make `rasa.production.url` a default env var in addition to these changes, so that if they do want to mount a custom nginx config, they're not forced to hard-code the URL?

Reply:
following this convo, had the same question 👀

Reply:
I've added the `RASA_PRODUCTION_URL` variable for the nginx deployment.

Reply:
Should it also be what is used in this line?